lib/modules/manager/poetry/artifacts.ts

```diff
@@ -92,7 +94,7 @@ export function getPoetryRequirement(
   return null;
 }
 
-function getPoetrySources(content: string, fileName: string): PoetrySource[] {
```

Comment:
nope, this should be logged when file parsing failed, so users know which file can't be parsed.

btw: this variable is used!